ScoutSuite/__main__.py

```diff
@@ -177,7 +177,7 @@ def run(provider,
 
     print_info('Launching Scout')
 
-    print_info('Authenticating to cloud provider')
+    print_info('Authenticating to %s cloud provider' % provider.upper())
```

Comment:
Ideally we'd format the name, but that's only available once the `cloud_provider` object is instantiated.

Reply:
I just added this minor change to make it easier when calling ScoutSuite against a cloud provider or if the output of the tool is later used somehow.
You can discard this change if you find it's not necessary.